### DISCUSSION TITLE
Comment utiliser CSS & JS externe ?

### DISCUSSION THREAD

**Discussion by Eliophot** [AmauriC/tarteaucitron.js#739] [Q&A]
Hello,
J'ai vu qu'en mode pro, il y a deux options (expert) pour modifier le CSS et le JS.
Comment cela fonctionne ?
Merci

**Comment by AmauriC** [comment]
Les options sont aussi dispo en v gratuite:
```
          "useExternalCss": false, /* If false, the tarteaucitron.css file will be loaded */
          "useExternalJs": false, /* If false, the tarteaucitron.js file will be loaded */
```

Si vous les activez, le fichier .css et les fichiers .js services et lang ne sont pas chargé via tarteaucitron.

C'est pratique si vous voulez faire de grosses modifs sur la librairie, ce n'est en général pas utile ; )

**Reply by ben212001** [reply]
Bonjour, 
Ou sont a placer ces 2 lignes de code svp (que ce soit pour l'installation pro ou l'installation manuelle gratuite) ?J'imagine que cela se passe à l'intérieur des balises <script> qu'il faut ajouter à chacun des sites que l'on veut doter de tarteaucitron ?

**Reply by AmauriC** [reply]
En version gratuite c'est dans l'initialisation du script, voir l'étape 1 de l'installation https://tarteaucitron.io/fr/install/
En v pro c'est depuis la console https://tarteaucitron.io/fr/pro/ dans les options.

**Reply by ben212001** [reply]
très bien merci ! en réalité, j'avais mal posé ma question: en version rpo, dans le cas où je coche la case "utiliser un fichier css externe" dans la console, ou dois-je indiquer le chemin vers le nouveau fichier css ? Je ne vois pas d'endroit dédié à côté de la case à cocher...

**Reply by AmauriC** [reply]
Il n'y a pas d'endroit dans la console, il vous suffit simplement d'ajoute le CSS quelque part sur votre site ;)

**Comment by Eliophot** [comment]
Ok donc en gros, en cochant le css, par exemple, je dois chargé mon propre css sur le site. Compris.
Effectivement pour une petite surcharge c'est un peu too much.
Le soucis, c'est qu'en mobile par exemple, on a l'icône qui se positionne mal en z-index ou qui se charge au dessus du footer. J'ai essayé de modifié le css dans la customisation du thème, mais sans effet.
Je pense que je vais faire une surcharge dans le fichier style du site ;)

**Comment by AmauriC** [comment]
Même en ajoutant `html body #tarteaucitronRoot` avant la règle ou en utilisant `!important` ?

**Reply by Eliophot** [reply]
Juste !important, mais ça aurait du faire le job, j'essaierai de faire un combo des deux :)
